src/client/mongodb/repository/BaseMongooseRepositoryImpl.ts

```
@@ -164,7 +164,7 @@ export class BaseMongooseRepositoryImpl<T extends object>
       const result = await this.model.deleteMany(conditions).lean();
       return result as unknown as any[];
     } catch (error) {
-      throw new Error('Error deleting documents');
+      throw error;
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)